website/manual.md

```diff
@@ -160,6 +160,9 @@ git clone --recurse-submodules https://github.com/denoland/deno.git
 Now we can start the build:
 
 ```bash
+# Initialize third party modules
+git submodule update --init
```

Comment:
It's covered above with `git clone --recurse-submodules`. 

Maybe we should automatically run `git submodules update --init` during `cargo build` if one of the submodule directories is empty...

Reply:
I guess adding this to [setup.py before downloading the binaries here](https://github.com/denoland/deno/blob/master/tools/setup.py#L24) should fix this.

```diff
diff --git a/tools/setup.py b/tools/setup.py
index e18b5ffd..5290f8d4 100755
--- a/tools/setup.py
+++ b/tools/setup.py
@@ -22,6 +22,9 @@ def main():
 
     args = parser.parse_args()
 
+    if not os.path.exists(third_party_path):
+        run(["git", "submodules", "update", "--init"])
+
     if args.no_binary_download:
         print "no binary download"
     else:
```

Reply:
> Maybe we should automatically run `git submodules update --init` during `cargo build` if one of the submodule directories is empty...

This would be helpful
I've created an issue at https://github.com/denoland/deno/issues/3079, and closing this PR

Reply:
I think third_party_path will get created, so maybe this

```
    if not os.path.exists(os.path.join(third_party_path, "v8")):
        run(["git", "submodules", "update", "--init"])
```